node/forest_libp2p/src/service.rs

```diff
@@ -317,13 +315,11 @@ where
                         NetworkMessage::BitswapRequest { cid, response_channel } => {
                             if let Err(e) = swarm_stream.get_mut().want_block(cid.clone(), 1000) {
                                 warn!("Failed to send a bitswap want_block: {}", e.to_string());
-                            } else {
-                                if let Some(chans) = self.bitswap_response_channels.get_mut(&cid) {
-                                    chans.push(response_channel);
+                            } else if let Some(chans) = self.bitswap_response_channels.get_mut(&cid) {
+                                chans.push(response_channel);
                                 } else {
-                                    self.bitswap_response_channels.insert(cid, vec![response_channel]);
+                                self.bitswap_response_channels.insert(cid, vec![response_channel]);
                                 }
```

Comment:
could you fix all the mucked up indentation here?

Reply:
320 and 322 just need to be unindented